server/error-middleware.js

```diff
@@ -5,6 +5,11 @@ function errorMiddleware(err, req, res, next) {
     res.status(err.status).json({
       error: err.message
     });
+  } else if (err.code === '23503') {
+    console.error(err);
+    res.status(404).json({
+      error: "This isn't the pin you're looking for..."
+    });
```

Comment:
This isn't really telling me why it's here. What error scenario are you trying to handle? I'm sure there's a better, more direct way to address it.